yarn-project/end-to-end/src/e2e_token_contract/transfer.test.ts

```diff
@@ -0,0 +1,85 @@
+import { AztecAddress, CompleteAddress } from '@aztec/aztec.js';
```

Comment:
I just copied these test cases from `transfer_private` so does not need to be reviewed.